src/common/app-paths/directory-for-binaries/directory-for-binaries.injectable.ts

```diff
@@ -3,14 +3,18 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import { getInjectable } from "@ogre-tools/injectable";
-import path from "path";
 import directoryForUserDataInjectable from "../directory-for-user-data/directory-for-user-data.injectable";
+import getAbsolutePathInjectable from "../../path/get-absolute-path.injectable";
 
 const directoryForBinariesInjectable = getInjectable({
   id: "directory-for-binaries",
 
-  instantiate: (di) =>
-    path.join(di.inject(directoryForUserDataInjectable), "binaries"),
```

Comment:
Given such simple injectables I think that having them so deeply nested hurts readability on the use of them side.

Namely, the import paths get needlessly long.

I would like to propose that these get move to `src/common/app-paths/directory-for-binaries.injectable.ts` instead, especially since I doubt that they will ever have suplimental injectables. And even then those could be in a folder while this one is left in the parent folder.

Reply:
I sort of like having them separate especially since we have already started to compose upon them. And this sort of change should be simple to do we auto-refactorings.